modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/PlannerTest.java

```diff
@@ -428,6 +430,69 @@ public IgniteDistribution distribution() {
         assertEquals(3, plan.fragments().size());
     }
 
+    /** Tests bounds merge. */
+    @Test
+    public void testBoundsMerge() throws Exception {
```

Comment:
Does it make sense to check condition like "col1 IN NOT NULL AND col1 > 1"?
The plan may differs for indices with different null ordering.

Reply:
unfortunately for now it`s hard to make a stable test with different index usage, we can of course make a different table and DESC index but seems it didn`t need to change a bound ranges. I check all cases manually and it seems work well.